src/features/search/queries/tag.ts

```diff
@@ -0,0 +1,29 @@
+import type { UseInfiniteQueryOptions, UseInfiniteQueryResult } from '@tanstack/react-query';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import type { Pagination } from 'src/api/models/pagination';
+import type { TagQueryOptions } from 'src/api/models/queryOptions/tag/tagQueryOptions';
+import type { Tag } from 'src/api/models/tag/tag';
+import { TagService } from 'src/api/services/tagService';
+
+/** Gets base options. */
+export const getBaseTagsQueryOptions = ({ search = '' }: TagQueryOptions): UseInfiniteQueryOptions<Pagination<Tag>, Error> => ({
```

Comment:
Do we need to export it? Is it possible could be used somewhere else?

Reply:
It's used for prefetching tags in `getServerSideProps`